### DISCUSSION TITLE
Documentation: update description on how to setup AWS IoT Core credentials authentication

### DISCUSSION THREAD

**Discussion by massi-ang** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1704] [Q&A]
The instructions on how to setup authentication with AWS IoT Core are outdated and do not correspond to the source code.

**Comment by niyatim23** [comment]
Hi @massi-ang, the functions `createLwsIotCredentialProvider` and `freeIotCredentialProvider` mentioned in the readme and the sample application are a part of the [KVS Producer C SDK](https://github.com/awslabs/amazon-kinesis-video-streams-producer-c) which is one of the dependencies in this project. That could be the reason, you may not have found it by search before building the project. Is there something in specific that you are looking for?

**Reply by massi-ang** [reply]
The README of this project [has a section](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#setup-iot)  on enabling the AWS IoT integration for the authentication. These instructions are incomplete and outdated. I was not able to enable the integration by just following what is mentioned in the README. 

The README of the [KVS Producer C SDK](https://github.com/awslabs/amazon-kinesis-video-streams-producer-c) does not mention anything about this integration either. How is a developer supposed to find out that the SDK supports this authentication mode and how to enable it?

**Comment by disa6302** [comment]
@massi-ang , 
Can you please clarify what the issue is? We have not modified anything with the IoT set up since this doc was put together, more information on what step you are facing an issue with will help :)

The instructions in this page should work: https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/how-iot.html. If it does not, please clarify the steps you are following and the end issue you are facing.
